src/app/pages/home/onboarding/index.js

```diff
@@ -0,0 +1,160 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect, useDispatch } from '@wordpress/data';
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { BWAButton as Button } from '@app/components/atoms';
+import comment from '@app/assets/comment.png';
+import computer from '@app/assets/computer.png';
+import rocket from '@app/assets/rocket.png';
+import './style.scss';
+
+export default function Onboarding() {
+
+	const baseUrl = useSelect((select) => {
+		return select('bluehost/plugin').getAdminUrl();
+	}, []);
+
+	const {toggleSetting} = useDispatch('bluehost/plugin');
+
+	const launchHandler = () => {
+		toggleSetting('comingSoon');
```

Comment:
Should we reload the page after firing this? We could show some kind of banner or toast notification, but the `Coming Soon` label on the admin menu remains, so when you click `Launch Site`, it looks like nothing happens.

Reply:
I added code to remove that element from the admin bar, if present when launching the site.